OpenChain Telco SBOM Specification.md

```diff
@@ -29,7 +29,7 @@ OpenChain means [OpenChain ISO/IEC 5230:2020](https://www.iso.org/standard/81039
 ## 3.	Requirements
 
 ### 3.3	Data Format 
-A Telco Standard SBOM shall adhere to the version 2.2 of the SPDX Data Format as standardized in ISO/IEC 5962:2021 and as further described below with respect to the included elements. 
+A Telco Standard SBOM shall (unless othervise specified) adhere to the version 2.2 of the SPDX Data Format as standardized in ISO/IEC 5962:2021 and as further described below with respect to the included elements. 
```

Comment:
Typo: "otherwise".

Reply:
I wonder if this is ambiguous - does "otherwise specified" mean "elsewhere in this Telco spec", or "by the creator of a given SBOM"?

I suggest that any flexibility here is only towards allowing newer versions than 2.2, not permitting older versions.